tests/metrics/test_rank_metrics.py

```diff
@@ -0,0 +1,44 @@
+# -*- encoding: utf-8 -*-
+# @Time    :   2020/12/21
+# @Author  :   Zhichao Feng
+# @email   :   fzcbupt@gmail.com
+
+
+import os
+import sys
+import unittest
+
+sys.path.append(os.getcwd())
+import numpy as np
+from recbole.evaluator.metrics import metrics_dict
+
+
+class TestCases(object):

```

Comment:
Maybe, `pos_rank_sum` can‘t ensure your metric is right, please test the function `evaluator.collect` to ensure your `pos_rank_sum` is right.